js/project-page-functions.js

```diff
@@ -112,8 +112,14 @@ function onProjectPageLoaded() {
     }
   });
   var $body = $('body');
-  $body.scrollspy({target: '#right-sidebar-nav', offset: navHeight});
-  $body.scrollspy({target: '#left-sidebar-nav', offset: navHeight});
+  $body.scrollspy({'target': '#right-sidebar-nav', 'offset':navHeight});
+  // Offset in the above for some reason doesn't work, so we fix it this way with a little hack:
```

Comment:
@richmahn I submitted a PR on your fork that adds unit testing for this code. https://github.com/richmahn/door43.org/pull/2